components/rating.css

```diff
@@ -0,0 +1,43 @@
+:root{
+    --star-color: #ff9800;
+}
+
+.rating-card{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem 2rem;
+    margin: 1rem auto;
+    box-sizing: border-box;
+    flex-direction: column;
+    box-shadow: 1px 1px 10px 10px rgba(0,0,0,0.1);
+    border-radius: 12px;
+    line-height: 1.5;
+}
+
+.rating-heading{
+    font-size: 1.2rem;
+}
+
+.rating-stars-container{
+    display: flex;
+    user-select: none;
+}
+
+.star{
+    font-size: 3rem;
+    color: var(--star-color);
+}
+
+.star:hover{
+    cursor: pointer;
+}
+
+.rating-iframe{
```

Comment:
just a suggestion but your ui would look a lot better if your iframe is centered

Reply:
Hey, thanks for the suggestion, I will work on that.